fog-aliyun.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'memory_profiler'
+  spec.add_development_dependency 'aliyun-sdk'
```

Comment:
I think the aliyun sdk should be regular dependency, as it used not only for dev scenarios

Reply:
I agree. But I think it can be  support in the next iterative process. Now, please check this way can meet your requirements.